src/index.ts

```diff
@@ -40,25 +45,30 @@ export const select = async (
   // Select the items we care about
   for (const option of options) {
     focus(input);
-    await findByText(container, option);
-    fireEvent.click(getByText(container, option));
+    const elementsMatchingText = await findAllByText(container, option);
```

Comment:
I'm getting several elements when searching for regexp: the option itself and the aria-context too. This is the best way I found to get the option element for now: get the last item as that's the option. Ideally I would like to use the options dropdown element as a container but I didn't figure out how to do that.

Reply:
Oh, I see! 

I think we might be able to work around this by only selecting accessible element - I'll take a look